CHANGES.txt

```diff
@@ -57,6 +57,10 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Add support for Python 3.13 (as of alpha 2). So far only affects
       expected bytecodes in ActionTests.py.
 
+  From Thaddeus Crews:
```

Comment:
Additions to CHANGES.txt are meant to  be alphabetical by last name, Please update.